requirements.txt

```diff
@@ -1,13 +1,14 @@
+alembic
+click
```

Comment:
I also added this in since some tools require it, python modules are cheap these days :)